canarytokens/aws_infra/operations.py

```diff
@@ -137,37 +148,40 @@ def _build_operation_payload(
     return payload
 
 
-# TODO: add handle exist for token validation
 async def get_handle_response(handle_id: str, operation: AWSInfraOperationType):
     """
     Check if a response has been added to the specified handle in the redis DB and return it.
     """
     handle = queries.get_aws_management_lambda_handle(handle_id)
+    default_error = AWSInfraServiceError.REQ_HANDLE_INVALID
+    default_error_message = service_error_map.get(default_error)
 
     if not handle:
-        logging.error(f"Handle with ID {handle_id} not found.")
-        return AWSInfraHandleResponse(handle=handle_id, message="Handle not found.")
+        return AWSInfraHandleResponse(
+            handle=handle_id,
+            message=default_error,
+            error=default_error_message,
```

Comment:
Here and below, are these the wrong way around?
```suggestion
            message=default_error_message,
            error=default_error,
```